### PR TITLE
*/maintain: fix dead link in slow query

### PR DESCRIPTION
**Pull request by dcalvin**
### What is changed, added or deleted? 

This PR fixed a dead link for Identify Slow Query

### Which version does your change affect? 

TiDB 2.1/3.0/Dev
 
### Checklist 

- [ ] Add a new file to `TOC.md`
- [ ] No Tab spaces anywhere 
- [ ] Leave a blank line both before and after a code block
- [ ] Keep the first level heading consistent with `title` in metadata
- [ ] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

File: dev/how-to/maintain/identify-slow-queries.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ category: how-to
 
 # 慢查询日志
 
-TiDB 在 V2.1.8 之后更改了慢日志格式，V2.1.8 之前的版本请看[这个文档](<https://github.com/pingcap/docs-cn/blob/master/v2.1/sql/slow-query.md>)。
+TiDB 在 V2.1.8 之后更改了慢日志格式，V2.1.8 之前的版本请看[这个文档](<https://github.com/pingcap/docs-cn/blob/master/v2.1-legacy/sql/slow-query.md>)。
```
Comment (lilin90) [inline dev/how-to/maintain/identify-slow-queries.md]:
```suggestion
TiDB 在 V2.1.8 之后更改了慢日志格式，V2.1.8 之前的版本请看[这个文档](https://github.com/pingcap/docs-cn/blob/master/v2.1-legacy/sql/slow-query.md)。
```

File: v2.1/how-to/maintain/identify-slow-queries.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ category: how-to
 
 # 慢查询日志
 
-TiDB 在 V2.1.8 之后更改了慢日志格式，V2.1.8 之前的版本请看[这个文档](<https://github.com/pingcap/docs-cn/blob/master/v2.1/sql/slow-query.md>)。
+TiDB 在 V2.1.8 之后更改了慢日志格式，V2.1.8 之前的版本请看[这个文档](<https://github.com/pingcap/docs-cn/blob/master/v2.1-legacy/sql/slow-query.md>)。
```
Comment (lilin90) [inline v2.1/how-to/maintain/identify-slow-queries.md]:
```suggestion
TiDB 在 V2.1.8 之后更改了慢日志格式，V2.1.8 之前的版本请看[这个文档](https://github.com/pingcap/docs-cn/blob/master/v2.1-legacy/sql/slow-query.md)。
```

File: v3.0/how-to/maintain/identify-slow-queries.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ aliases: ['/docs-cn/sql/slow-query/']
 
 # 慢查询日志
 
-TiDB 在 V2.1.8 之后更改了慢日志格式，V2.1.8 之前的版本请看[这个文档](<https://github.com/pingcap/docs-cn/blob/master/v2.1/sql/slow-query.md>)。
+TiDB 在 V2.1.8 之后更改了慢日志格式，V2.1.8 之前的版本请看[这个文档](<https://github.com/pingcap/docs-cn/blob/master/v2.1-legacy/sql/slow-query.md>)。
```
Comment (lilin90) [inline v3.0/how-to/maintain/identify-slow-queries.md]:
```suggestion
TiDB 在 V2.1.8 之后更改了慢日志格式，V2.1.8 之前的版本请看[这个文档](https://github.com/pingcap/docs-cn/blob/master/v2.1-legacy/sql/slow-query.md)。
```

**Comment by dcalvin** [comment]
@lilin90  PTAL again. Thanks.

**Pull request merged**